tests/integration/src/test/scala/mesosphere/marathon/integration/facades/MesosFacade.scala

```diff
@@ -116,7 +116,7 @@ class MesosFacade(val url: String, val waitTime: FiniteDuration = 30.seconds)(im
 
   def state: RestResult[ITMesosState] = {
     logger.info(s"fetching state from $url")
-    result(requestFor[ITMesosState](Get(s"$url/state.json")), waitTime)
+    result(requestFor[ITMesosState](Get(s"$url/state")), waitTime)
```

Comment:
we need to talk about this...   `/state` is required now with Mesos 1.8.   It was introduced with an earlier version of mesos.   Do we require all devs to switch to a version of mesos that works for `/state` or do we detect and switch from `/state.json` and `/state` dependent on version?

Reply:
`/state` is supported in older Mesos versions so we can just leave it like this